README.md

```diff
@@ -5,25 +5,29 @@
 
 ## TimescaleDB
 
-TimescaleDB is an open-source database designed to make SQL scalable for
-time-series data. It is engineered up from PostgreSQL, providing automatic
-partitioning across time and space (partitioning key), as well as full
-SQL support.
-
-TimescaleDB is packaged as a PostgreSQL extension.  All code is licensed
-under the Apache-2 open-source license, with the exception of source code
-under the `tsl` subdirectory, which is licensed under
-the [Timescale License (TSL)](https://github.com/timescale/timescaledb/blob/master/tsl/LICENSE-TIMESCALE).
-For clarity, all code files reference licensing in their header.  [Contributors welcome.](https://github.com/timescale/timescaledb/blob/master/CONTRIBUTING.md)
+TimescaleDB is an open-source database designed to make SQL scalable for time-series data. 
+It is engineered up from PostgreSQL and packaged as a PostgreSQL extension, providing automatic 
+partitioning across time and space (partitioning key), as well as full SQL support.
+
+[Timescale Cloud](https://tsdb.co/GitHubTimescaleCloud) is our fully managed, hosted version of 
+TimescaleDB, available in the cloud of your choice (pay-as-you-go, $300 in credits to start).
```

Comment:
Do we want to mention a specific sum in the readme? Is it not better to just refer to a page with details?

Reply:
I don't feel strongly either way and am fine to edit to (pay-as-you-go, with free trial credits to start).